### ISSUE TITLE
LLOQ values show up wrongly as filled points if PK-sim function "copy to clipboard is used

### ISSUE THREAD

**Issue by tobiasK2001** [Open-Systems-Pharmacology/PK-Sim#1635]
they should show up as open circles

![image](https://user-images.githubusercontent.com/25585248/106249051-1e2b0000-6212-11eb-8cbf-0b0a81435d0b.png)

In PK_sim chart the points below LLOQ are shown as open circles and here is a screenshot giving an impression how it should look.
![image](https://user-images.githubusercontent.com/25585248/106249087-2b47ef00-6212-11eb-92c3-e00ab3491ead.png)

Not a big thing, but there seems to be a bug when the chart is copied via the clipboard. I am using PK-Sim 9.1
Same thing if I copy the Chart from The Parameter identification 
![image](https://user-images.githubusercontent.com/25585248/106249551-ca6ce680-6212-11eb-8dca-3a8cff014c55.png)

**Comment by AyaSaleh90** [comment]
Dear @tobiasK2001, can you tell me please how you got this horizontal line for the value of LLOQ level? is it done in the PK-Sim itself?

**Comment by tobiasK2001** [comment]
yes, this is done by PK-Sim if import values as "<1" given that the unit of the concentration values is nM in that case. Than PK-sim recognizes that 1 nM is the LLoQ and draws the lines to the data

**Comment by AyaSaleh90** [comment]
> yes, this is done by PK-Sim if import values as "<1" given that the unit of the concentration values is nM in that case. Than PK-sim recognizes that 1 nM is the LLoQ and draws the lines to the data

Thank you so much

**Comment by AyaSaleh90** [comment]
> yes, this is done by PK-Sim if import values as "<1" given that the unit of the concentration values is nM in that case. Than PK-sim recognizes that 1 nM is the LLoQ and draws the lines to the data

First thanks for the reply,, but let me more specific since I'm not an expert in PK-Sim, now my LLOQ values is 0.25 pg/mL, so upon importing the dataset I should adjust them to ''<0.25'', so PK-sim can recognize them??

**Comment by tobiasK2001** [comment]
in short yes! But I cannot explain it better than here:
https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/import-edit-observed-data
marking data as LLOQ is not just cosmetic as it can impact your in PI as described here 
https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/parameter-identification#handling-of-lloq-values

and there a lot more good stuff in the manual there. Enjoy reading!

**Comment by AyaSaleh90** [comment]
@tobiasK2001  I really appreciate your help

**Comment by msevestre** [comment]
@tobiasK2001  Thanks for reporting .Looks like a bug in the chart component we are using.

**Comment by Yuri05** [comment]
Stil an issues in V11.1
